### ISSUE TITLE
Insufficient instructions and error details when creating a password. Labels or Instructions

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1465]
## Description of issue

When creating an account the instructions for the password field indicate that the password “must be at least 8 characters and hard to guess” without specific information about what makes a password hard to guess. Consider more specific instructions, particularly if validation for the field is based on defined criteria. 

<img width="772" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeEtZQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5b3f0512efc0b011c9e09a1e50b72e51c7cf5eaf/image.png" alt="image.png" />

On the Account Creation page, the password requirements are not specific. Certain strings (examples: 11111111, a1a1a1a1a1, aaaaaaaaa, abcdefg) are returned with the error “Choose a password that is harder to guess” but no specific instructions are provided. Depending on how the field is validated, suggestions such as “use a combination of capital and lower case letters, numbers and special characters” could help users pick a stronger password.

<img width="776" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeFdZQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--fd695bef6024532245c7f11899b87a6c94b8264f/image.png" alt="image.png" />

## Potential fix

- Use the same content we designed for resetting a password. 
- Create a more direct error message for simple password
- Suggest passphrases?

**Comment by amazingphilippe** [comment]
This has been fixed: <img width="1480" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK2I0QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--41334764b93f37484783ef0183c5b077af01fdd6/Capture%20d%E2%80%99e%CC%81cran%2C%20le%202024-03-20%20a%CC%80%2015.19.30.png" alt="Capture d’écran, le 2024-03-20 à 15.19.30.png" />
